### PR TITLE
Adding documentation

### PR DESCRIPTION
**Pull request by ahmadnazeri**
# Description

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: docs/getting-started/serving-for-inference-and-training.md [hunk 1]
```diff
@@ -12,6 +12,27 @@ To serve a feature we initialize a serving client, specify the names and variant
...
+
+```python
+import featureform as ff
+
+@ff.ondemand_feature(name="sex_as_int", variant="quickstart")
+def sex_as_int(serving_client, params, entities):
+    passenger_id = params[0]
+    df = params[1]
+    df['Sex'] = df['Sex'].map( {'female': 0, 'male': 1} ).astype(int)
+    return df.loc[df["PassengerId"] == passenger_id]["Sex"].values
```
Comment (sdreyer) [inline docs/getting-started/serving-for-inference-and-training.md:27]: Does this return a list?

Comment (ahmadnazeri) [reply docs/getting-started/serving-for-inference-and-training.md:27]: it would be numpy.ndarray of size 1.

Comment (sdreyer) [reply docs/getting-started/serving-for-inference-and-training.md:27]: It should just return a scalar right?

**Review by MMBazel — APPROVED** [review]
Looks good! Thanks @ahmadnazeri !

File: docs/getting-started/serving-for-inference-and-training.md (hunk 1 above)
Comment (ahmadnazeri) [reply docs/getting-started/serving-for-inference-and-training.md:27]: it works fine as is but I will change it to scalar

**Review by sdreyer — APPROVED** [review]

**Pull request merged**